modules/nf-core/mageck/test/tests/main.nf.test

```diff
@@ -0,0 +1,75 @@
+nextflow_process {
+
+    name "Test Process MAGECK_TEST"
+    script "../main.nf"
+    process "MAGECK_TEST"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/test"
+
+    test("mageck-test - csv") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                 [ id:'test', single_end:false ], // meta map
+                 file(params.test_data['mus_musculus']['csv']['count_table'], checkIfExists: true)
+             ]
+                """
+            }
+        }
+
+                then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() },
```

Comment:
```suggestion
```
if they are unstable as mentioned below, snapshotting the complete output will not work. See https://hackmd.io/@nf-core/ryWsFLwyT for example of alternative assertions for the unstable ones